src/components/utils/index.tsx

```diff
@@ -0,0 +1,59 @@
+export const number: { num: string }[] = [
+  { num: "1" },
+  { num: "2" },
+  { num: "3" },
+  { num: "4" },
+  { num: "5" },
+  { num: "6" },
+  { num: "7" },
+  { num: "8" },
+  { num: "9" },
+  { num: "0" },
+  { num: "A/C" },
+  { num: "(" },
+  { num: "." },
+  { num: "+/-" },
+  { num: "+" },
+  { num: "-" },
+  { num: "*" },
+  { num: "/" },
+  { num: "%" },
+  { num: ")" },
+  { num: "=" },
+];
+
+export const operator = (): string[] => {
+  const operator = number
```

Comment:
Here also consider adding s to array name

Reply:
yes done